### ISSUE TITLE
Electron apps based are not accessible via MacOS "Hover Text" unless Voice Over is turned on

### ISSUE THREAD

**Issue by rfairweather** [electron/electron#25695]
**Issue: Electron apps based are not accessible via Hover Text unless Voice Over is turned on.**

### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details
* **Electron Version:**
  * All versions
* **Operating System:**
  * MacOS Catalina
* **Last Known Working Electron version:**
  * None

### Expected Behavior
Hover Text appears for the text in the Electron app

### Actual Behavior
Hover Text does not appear (but it does on Chromium)

### To Reproduce
1. Install any Electron app.
2. Launch the Electron app.
3. In MacOS System Preferences, go to Accessibility > Zoom, and enable Hover Text.
4. Hover over text in the Electron app.

Now observe what happens when we enable Voice Over...

5. Turn on Voice Over
6. Hover over text in the Electron app.
7. Observe that Hover Text shows for the text in the Electron app now that Voice Over is enabled

### Additional resources
https://support.apple.com/guide/mac-help/use-hover-text-mchlb203bc78/mac#:~:text=%20%20%201%20On%20your%20Mac%2C%20choose,menu%2C%20then%20choose%20a%20location.%20If...%20More%20
